executor/runtime/docker/docker_linux.go

```diff
@@ -44,10 +44,16 @@ const (
 	systemServiceStartTimeout = 90 * time.Second
 	umountNoFollow            = 0x8
 	sysFsCgroup               = "/sys/fs/cgroup"
+	hybridMountpoint          = "/sys/fs/cgroup/unified"
 	runcArgFormat             = "--root /var/run/docker/runtime-%s/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE %s %s"
 	defaultOomScore           = 1000
 )
 
+var (
+	isHybridOnce sync.Once
```

Comment:
Since this is a global variable, can you give it a more descriptive name?

Reply:
Done